python/ray/tests/test_object_spilling.py

```diff
@@ -278,34 +314,42 @@ def depends(arg):
 def test_spill_objects_automatically(object_spilling_config, shutdown_only):
     # Limit our object store to 75 MiB of memory.
     ray.init(
+        num_cpus=1,
         object_store_memory=75 * 1024 * 1024,
         _system_config={
-            "max_io_workers": 4,
+            "max_io_workers": 8,
             "automatic_object_spilling_enabled": True,
             "object_store_full_max_retries": 4,
             "object_store_full_initial_delay_ms": 100,
             "object_spilling_config": object_spilling_config,
+            "min_spilling_size": 30 * 1024 * 1024,
```

Comment:
Can you copy this to a new test instead of overwriting the old one? We should test both the fusion and no-fusion cases.